modules/oe_content_tender/config/install/core.entity_view_display.node.oe_tender.teaser.yml

```diff
@@ -0,0 +1,72 @@
+langcode: en
```

Comment:
I dont see anything in the ticket about a teaser view

Reply:
Fixed by removing Teaser view mode